modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationChanger.java

```diff
@@ -670,14 +688,21 @@ private void validateConfiguration(SuperRoot curRoots, SuperRoot changes) {
 
     private ConfigurationStorageListener configurationStorageListener() {
         return changedEntries -> {
+            Map<String, ? extends Serializable> changedValues = changedEntries.values();
+
+            // We need to ignore deletion of deprecated values.
+            for (String ignoredValue : ignoredKeys) {
+                changedValues.remove(ignoredValue);
+            }
```

Comment:
Could you please explain what's going on here? I don't think that it is possible to receive deleted values from a storage notification, they were filtered-out before that

Reply:
I see, my bad. Should have realized that myself